azurerm/internal/services/servicebus/validate/servicebus_namespace.go

```diff
@@ -0,0 +1,44 @@
+package validate
+
+import (
+	"fmt"
+	"regexp"
+)
+
+// validation
+func ServiceBusNamespaceName(v interface{}, k string) (warnings []string, errors []error) {
+	value := v.(string)
+
+	if !regexp.MustCompile("[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}").MatchString(value) {
```

Comment:
I think we have a uuid regex helper somewhere, but we can also leverage `validation.IsUUID()`